src/app/body/task-details/task-details.component.html

```diff
@@ -50,24 +50,28 @@ <h2>Task Details</h2>
-                                    <a *ngIf="prLinked" data-target="#gitPrModel" data-toggle="modal" title="Select PR" class="btn btn-status" data-placement="top" (click)="linkPr()">
+                            <div >
+                                <ng-container *ngIf="provider === 'Github'">
+                                  <li class="nav-item">
+                                    <span class="btn">
+                                      <img src="../../../assets/git.png" height="25" class="rounded-circle" width="25" alt="Git icon" data-toggle="modal" title="Select PR" data-target="#gitPrModel" (click)="linkPr()">
+                                      <a *ngIf="prLinked" data-target="#gitPrModel" data-toggle="modal" title="Select PR" class="btn btn-status" data-placement="top" (click)="linkPr()">
                                         PR status: <span class="text-capitalize" style="color: green;">{{prData.state}}</span>
-                                    </a>
-                                </span>
-                            </li>
-                            <li class="nav-item" *ngIf="githubRepoExists">
-                                <span class="btn">
-                                    <img src="../../../assets/gitlabIcon.svg" *ngIf="!prLinked" height="25" class="rounded-circle" width="25"
-                                    alt="user" data-toggle="modal" title="Select PR" data-target="#gitlabPrModel" (click)="linkPr()">
-                                    <a *ngIf="prLinked" data-target="#gitlabPrModel" data-toggle="modal" title="Select PR" class="btn btn-status" data-placement="top" (click)="linkPr()">
+                                      </a>
+                                    </span>
+                                  </li>
+                                </ng-container>
+                                <ng-container *ngIf="provider === 'gitlab'">
+                                  <li class="nav-item">
+                                    <span class="btn">
+                                      <img src="../../../assets/gitlabIcon.svg" height="25" class="rounded-circle" width="25" alt="Gitlab icon" data-toggle="modal" title="Select PR" data-target="#gitlabPrModel" (click)="linkPr()">
+                                      <a *ngIf="prLinked" data-target="#gitlabPrModel" data-toggle="modal" title="Select PR" class="btn btn-status" data-placement="top" (click)="linkPr()">
                                         PR status: <span class="text-capitalize" style="color: green;">{{prData.state}}</span>
-                                    </a>
-                                </span>
-                            </li>
+                                      </a>
+                                    </span>
+                                  </li>
+                                </ng-container>
+                            </div>
                             <!-- <li class="nav-item"  *ngIf="githubRepoExists">
```

Comment:
Can this commented part removed?

Reply:
Not sure with removing this as it contains a dropdown to show pr details , It is kept as commented for future implementation as per my knowledge.